src/app/api/services/dot-license/dot-license.service.ts

```diff
@@ -27,6 +76,28 @@ export class DotLicenseService {
         return this.getLicense().pipe(map((license) => license['level'] >= 200));
     }
 
+    canAccessEnterprisePortlet(url: string): Observable<boolean> {
+        return this.isEnterprise().pipe(
+            take(1),
+            map((isEnterpriseUser: boolean) => {
+                const urlMatch = this.checksIfEnterpriseUrl(url);
+                return urlMatch ? urlMatch && isEnterpriseUser : true;
```

Comment:
```suggestion
                return urlMatch && isEnterpriseUser || true;
```

Reply:
in your suggestions if `urlMatch=true` and `isEnterpriseUser=false` then it returns `true`, which is not ok,

in the previous implementation it returns `false`